docs/source/guides/configuration.rst

```diff
@@ -54,3 +55,6 @@ Environment-specific parameters
 - ``load_delay`` is a waiting time in milliseconds before loading committed 
   block from next peer. We recommend setting this number the same value as 
   ``proposal_delay`` or even higher.
+- ``mst_enable`` enables or disables multisignature transaction support in
+  Iroha. We recommend setting this parameter to ``false`` at the moment until
```

Comment:
I would suggest removing the last sentence, as long it might be outdated soon and this point might be forgotten